app/src/lib/stats/stats-store.ts

```diff
@@ -427,6 +430,27 @@ export class StatsStore {
     }))
   }
 
+  /** Record that the user pushed to GitHub.com */
+  public async recordPushToGitHub(): Promise<void> {
+    return this.updateDailyMeasures(m => ({
+      dotcomPushCount: m.dotcomPushCount + 1,
```

Comment:
These counts are only incremented per-push, and not including the number of commits being pushed as part of this operation. This means a user pushing each commit would look very different to a person pushing at the end of the day, even if they push the same number of commits.

We know the number of commits the current branch is ahead before pushing, so do we care about the granularity of these pushes?

cc @telliott27

Reply:
I don't think we need to explicitly collect the number of commits per push. We separately track the number of commits made, so assuming all or most commits get pushed, we can estimate the number of commits per push with `numCommits / numPushes`